src/db.ts

```diff
@@ -367,13 +404,19 @@ export const saveDeviceChallenge = async (challenge) =>
     JSON.stringify(challenge, undefined, 2)
   );
 
-export const saveBooking = (accountId, booking) => {
-  return findPersonByAccount({ id: accountId })
-    .then((person) => {
-      person.transactions.push(booking);
-      return person;
-    })
-    .then(savePerson);
+export const saveBooking = async (accountId, booking) => {
+  let person;
+  const personId = await getPersonIdByAccount({ id: accountId });
+  const personLockKey = `redlock:${process.env.MOCKSOLARIS_REDIS_PREFIX}:person:${personId}`;
```

Comment:
just a tiny comment i saw this line repeated multiple time can we make a simple function to get it 🙏 
const  generatePersonLockKey = (personId)=> {
  return `redlock:${process.env.MOCKSOLARIS_REDIS_PREFIX}:person:${personId}`;
}

const personLockKey = generatePersonLockKey(personId);